key.py

```diff
@@ -0,0 +1,8 @@
+urlInstagram = "https://www.instagram.com/"
+
+APP_TOKEN = '11166443968.c11ca5e.3cbeb9018cd249a589cace56eaaff7e4'
```

Comment:
I can't find where this variable is using...

Reply:
deleted